Testing/SystemTests/tests/framework/MDNormBackgroundHYSPECTest.py

```diff
@@ -0,0 +1,323 @@
+
+        # Test
+        self.test_property_setup()
+
+        # Prepare background workspace
+        self.prepare_single_exp_info_background(input_md_name='reduced_1',
+                                                output_md_name='bkgd_md',
+                                                target_qframe='Q_lab')
+
+        # Test new solutions
+        r = MDNorm(InputWorkspace='merged',
+                   BackgroundWorkspace='bkgd_md',
+                   Dimension0Name='QDimension1',
+                   Dimension0Binning='-5,0.05,5',
+                   Dimension1Name='QDimension2',
+                   Dimension1Binning='-5,0.05,5',
+                   Dimension2Name='DeltaE',
+                   Dimension2Binning='-2,2',
+                   Dimension3Name='QDimension0',
+                   Dimension3Binning='-0.5,0.5',
+                   SymmetryOperations='x,y,z;x,-y,z;x,y,-z;x,-y,-z',
+                   OutputWorkspace='clean_data',
+                   OutputDataWorkspace='dataMD',
+                   OutputNormalizationWorkspace='normMD',
+                   OutputBackgroundNormalizationWorkspace='background_normMD')
+        assert hasattr(r, 'OutputBackgroundNormalizationWorkspace')
+        clean_data = r.OutputWorkspace
+
+        SaveMD(InputWorkspace=clean_data, Filename='/tmp/clean.nxs')
+        SaveMD(InputWorkspace='normMD', Filename='/tmp/sample_norm_round1.nxs')
+        SaveMD(InputWorkspace='dataMD', Filename='/tmp/sample_data_round1.nxs')
+        SaveMD(InputWorkspace='background_normMD', Filename='/tmp/normed_background_round1.nxs')
+        # SaveMD(InputWorkspace='background_dataMD', Filename='/tmp/background_data_round1.nxs')
+
+        # 2nd round
+        clean_data = self.normalize_with_background('sum', 'clean2', (12., 15.),
+                                                    sample_temp_ws_names=('dataMD', 'normMD'),
+                                                    background_temp_ws_names=('background_dataMD', 'background_normMD'))
+
+        # @JESSE: Here is the difference
+        # save MD for 2nd round
+        # save
+        SaveMD(InputWorkspace=clean_data, Filename='/tmp/clean_round2.nxs')
+        # SaveMD(InputWorkspace='normMD', Filename='/tmp/sample_norm_round1.nxs')
+        # SaveMD(InputWorkspace='dataMD', Filename='/tmp/sample_data_round1.nxs')
+        # SaveMD(InputWorkspace='background_normMD', Filename='/tmp/normed_background_round1.nxs')
+        # SaveMD(InputWorkspace='background_dataMD', Filename='/tmp/background_data_round1.nxs')
+
+        # @JESSE: If LATER you set log value range from (10, 12) to (10, 15) on line 115,
+        # Then clean_data (on line 152) shall be same as clean_data on line 160 calculated in the current test setup
+
+        assert mtd.doesExist('clean_data')
+
+    def validate(self):
+        self.tolerance = 1e-8
+        return 'result', 'MDNormHYSPEC.nxs'
```

Comment:
@jrmcgaha-dev You may return a 2-tuple as (list(), list()), for example, (['result1', 'result2), ['file1.nxs', file2.nxs'])
But you need to verify my guess.